test/SlowTests/Issues/RavenDB-13349.cs

```diff
@@ -94,9 +95,9 @@ public async Task Query_with_nested_JsonPropertyName_inside_select_clause(Option
                     var result = await query.ToListAsync();
 
                     Assert.Equal(3, result.Count);
-                    Assert.Equal("city1", result[0].CityName);
-                    Assert.Equal("city1", result[1].CityName);
-                    Assert.Equal("city2", result[2].CityName);
+                    Assert.Contains("city1", result.Select(i => i.CityName));
```

Comment:
Can we have another test that checks the order?
Or, you can modify the way we save the documents (save them one by one in a different session).

Reply:
What's the point? This test was not supposed to verify the order, wasn't it?